src/main/scala/nl/knaw/dans/easy/multideposit/package.scala

```diff
@@ -319,36 +319,47 @@ package object multideposit {
   val propsFileName = "deposit.properties"
   val springfieldActionsFileName = "springfield-actions.xml"
 
+  // mdDir/datasetID/
```

Comment:
These additions are purely for convenience. They show the abstract path of the `File` that each function returns.